packages/flutter_fuchsia/lib/flutter_fuchsia.dart

```diff
@@ -0,0 +1,12 @@
+// Copyright 2018 The Fuchsia Authors. All rights reserved.
```

Comment:
I think we want to use Flutter's license headers for all packages in this repo. /cc @Hixie